packages/viz/src/lib/Layer.ts

```diff
@@ -40,7 +41,10 @@ export class Layer implements StyledLayer {
 
   getMapInstance(): Deck {
     if (this._deckInstance === undefined) {
-      throw Error('Layer not attached to map');
+      throw new CartoLayerError(
+        'Layer not attached to map',
```

Comment:
I think we should put `Cannot return map instance because the layer has not been added to a map yet`.

Reply:
Done